dev/buildAll.gradle

```diff
@@ -78,6 +78,12 @@ def openApiCliFile = file(projDir + project.properties['codewind.core.prereqlib.
 
 def filePrefix = "codewind-openapi-eclipse"
 
+def swtBotEclipseFinderFile = file(projDir + project.properties['eclipse.target.platform.ies.folder'] + '/eclipse/dropins/plugins/org.eclipse.swtbot.eclipse.finder_2.8.0.201906121535.jar')
+def swtBotJunitFile = file(projDir + project.properties['eclipse.target.platform.ies.folder'] + '/eclipse/dropins/plugins/org.eclipse.swtbot.junit4_x_2.8.0.201906121535.jar')
+def swtBotSwtFile = file(projDir + project.properties['eclipse.target.platform.ies.folder'] + '/eclipse/dropins/plugins/org.eclipse.swtbot.swt.finder_2.8.0.201906121535.jar')
+
+
+
 
 // Download dependent plugins from maven repository for 'org.eclipse.codewind.core'
```

Comment:
Not part of your change, but could you change 'org.eclipse.codewind.core' to 'org.eclipse.codewind.openapi.core' ?